src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -244,9 +244,12 @@ function PaymentMethodList({
                 // The card should be grouped to a specific domain and such domain already exists in a assignedCardsGrouped
                 if (assignedCardsGrouped.some((item) => item.isGroupedCardDomain && item.description === card.domainName) && !isAdminIssuedVirtualCard) {
                     const domainGroupIndex = assignedCardsGrouped.findIndex((item) => item.isGroupedCardDomain && item.description === card.domainName);
-                    assignedCardsGrouped[domainGroupIndex].errors = {...assignedCardsGrouped[domainGroupIndex].errors, ...card.errors};
-                    if (card.fraud === CONST.EXPENSIFY_CARD.FRAUD_TYPES.DOMAIN || card.fraud === CONST.EXPENSIFY_CARD.FRAUD_TYPES.INDIVIDUAL) {
-                        assignedCardsGrouped[domainGroupIndex].brickRoadIndicator = CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR;
+                    const assignedCardsGroupedItem = assignedCardsGrouped.at(domainGroupIndex);
```

Comment:
Please, make sure it doesn't break anything, cause .at() accepts negative integers while [] not. See https://github.com/Expensify/App/pull/49534#discussion_r1772650621